common/src/main/java/tds/common/health/Health.java

```diff
@@ -0,0 +1,29 @@
+package tds.common.health;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@AutoValue
```

Comment:
Maybe we can discuss this in person.  I don't see a lot of value for the `AutoValue` for a class that has 2 fields.  In addition our team prefers immutable objects (except populated via frameworks) and I think this should follow that pattern.